variables.tf

```diff
@@ -231,6 +231,29 @@ variable "sysdig_crn" {
   default     = null
 }
 
+variable "force_delete" {
+  type        = bool
+  description = "Deletes all the objects in the COS Bucket before bucket is deleted."
+  default     = true
+}
+
+variable "single_site_location" {
+  type        = string
+  description = "Specify the single site bucket location. If you pass a value for this, ensure to set the value of var.region and var.cross_region_location to null."
+  default     = null
+
+  validation {
+    condition     = var.single_site_location == null || can(regex("ams03|che01|hkg02|mel01|mex01|mil01|mon01|osl01|par01|sjc04|sao01|seo01|sng01|tor01", var.single_site_location))
+    error_message = "Variable 'cross_region_location' must be 'ams03', 'che01', 'hkg02', 'mel01', 'mex01', 'mil01', 'mon01', 'osl01', 'par01', 'sjc04', 'sao01', 'seo01', 'sng01', 'tor01', or 'null'."
+  }
+}
+
+variable "hard_quota" {
+  type        = number
+  description = "Deletes all the objects in the COS Bucket before bucket is deleted."
```

Comment:
Wrong description? Make sure to also mention what happens when no value is passed for the variable.

Reply:
fixed